ush/machine/noaacloud.yaml

```diff
@@ -28,3 +28,10 @@ platform:
 data:
   ics_lbcs:
     FV3GFS: /contrib/GST/model_data/FV3GFS/${yyyymmdd}${hh}
+
+GSI-fix:
+  local:
+    output_path: ./sorc/gsi
+    file_path: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_gsi
+    file_names:
+      - gsi-fix.22.07.27.tar.gz
```

Comment:
removed this from the noaacloud machine file. added noaacloud FIXgsi location. also: added the remote gsi-fix file section to data_locations.yaml for generalized pulling of the gsi-fix files. 

@christinaholtNOAA do you want a FIXgsi location set for all the machines where we have the fix files staged? my understanding is that these files are pulled after building GSI in the srw build process, so not sure what is most useful here.

Reply:
to add the GSI fix files to the appropriate "UFS_SRW_App/develop/fix/" directories on each platform (aside from noaacloud which is already done), someone with sudo privileges to the following directories may need to perform:

Cheyenne:  sudo mv  /glade/work/epicufsrt/contrib/fix_gsi  /glade/p/ral/jntp/UFS_SRW_App/develop/fix/

Orion:  sudo mv /work/noaa/epic-ps/role-epic-ps/fix_gsi  /work/noaa/fv3-cam/UFS_SRW_App/develop/fix/

Hera:  sudo mv /scratch1/NCEPDEV/nems/role.epic/fix_gsi  /scratch2/BMC/det/UFS_SRW_App/develop/fix/

Jet:  sudo mv /mnt/lfs4/HFIP/hfv3gfs/role.epic/fix_gsi /mnt/lfs4/BMC/wrfruc/UFS_SRW_App/develop/fix/

Gaea:   sudo mv /lustre/f2/dev/role.epic/contrib/fix_gsi  /lustre/f2/pdata/ncep_shared/UFS_SRW_App/develop/fix/

maybe this can be done in a follow-up, short pr.